### PR TITLE
Merge pull request #89 from saraedum/sage-pins

### PR DESCRIPTION
**Pull request by saraedum**
this is the first part of https://github.com/conda-forge/sage-feedstock/issues/28

### CODE REVIEW

**Comment by saraedum** [comment]
@conda-forge-admin, please rerender.

**Comment by saraedum** [comment]
This is meant to be merged after #88; I only did the rerender here.

**Comment by saraedum** [comment]
@asmeurer, is that what you had in mind?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ source:
   sha256: 6f5d42b306a2c03073dd95086f80602bd1a0b929b8ad19c6d219c8ca8e96da26
 
 build:
-  number: 0
+  number: 2
```
Comment (asmeurer) [inline recipe/meta.yaml]: Why did you bump this to 2?

**Comment by asmeurer** [comment]
Yeah, assuming they are all correct.

**Comment by saraedum** [comment]
> Yeah, assuming they are all correct.

I copied them over, so they should be fine.

File: recipe/meta.yaml (hunk 1 above)
Comment (saraedum) [reply recipe/meta.yaml]: Because #88 bumps this to 1.

**Pull request merged**